### PR TITLE
net-firewall/fwbuilder: bump to 6.0.0_beta

### PR DESCRIPTION
**Pull request by nilathak**
Package-Manager: Portage-2.3.49, Repoman-2.3.10

### CODE REVIEW

File: net-firewall/fwbuilder/files/fwbuilder-6.0.0_beta-gcc-maybe-uninitialized.patch [hunk 1]
```diff
@@ -0,0 +1,52 @@
+From 1ec3d2315d6c1a8f020763cd7fa019c9b8ed2921 Mon Sep 17 00:00:00 2001
+From: Hannes Schweizer <nilathak@users.noreply.github.com>
+Date: Tue, 11 Sep 2018 21:18:10 +0200
+Subject: [PATCH] fix: demote GCC maybe-uninitialized error to warning again
+
+After enabling stricter compile warnings with the following commit
```
Comment (mgorny) [inline net-firewall/fwbuilder/files/fwbuilder-6.0.0_beta-gcc-maybe-uninitialized.patch]: `-Werror` is forbidden by Gentoo policy, so you should strip that instead.

File: net-firewall/fwbuilder/fwbuilder-6.0.0_beta.ebuild [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit autotools gnome2-utils qmake-utils eapi7-ver
```
Comment (mgorny) [inline net-firewall/fwbuilder/fwbuilder-6.0.0_beta.ebuild]: Please sort.

File: net-firewall/fwbuilder/fwbuilder-6.0.0_beta.ebuild [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+DESCRIPTION="A firewall GUI"
+HOMEPAGE="https://github.com/fwbuilder/fwbuilder"
+if [[ ${PV} == 9999* ]]
+then
+	EGIT_REPO_URI="https://github.com/fwbuilder/fwbuilder"
+	inherit git-r3
+	KEYWORDS=""
+else
+	MY_PV=$(ver_rs 3 '-')
+	S="${WORKDIR}/${PN}-$(ver_rs 3 '-')"
```
Comment (mgorny) [inline net-firewall/fwbuilder/fwbuilder-6.0.0_beta.ebuild]: Why not use `MY_PV` here?

File: net-firewall/fwbuilder/fwbuilder-6.0.0_beta.ebuild [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+	SRC_URI="https://github.com/fwbuilder/${PN}/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
```
Comment (mgorny) [inline net-firewall/fwbuilder/fwbuilder-6.0.0_beta.ebuild]: `-> ${PN}-${MY_PV}`, for consistency with path used in `S`.

File: net-firewall/fwbuilder/fwbuilder-6.0.0_beta.ebuild [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+	RESTRICT="mirror"
```
Comment (mgorny) [inline net-firewall/fwbuilder/fwbuilder-6.0.0_beta.ebuild]: Why? I can't see why GPL-2+ would prohibit mirroring.

**Comment by mgorny** [comment]
Ping.

**Comment by onkelbeh** [comment]
Please do it, changes take ~2 minutes.

**Comment by mgorny** [comment]
Closing due to no activity. Please reopen or file new when you are available again.

**Pull request closed without merging**